### PR TITLE
Various things

### PR DESCRIPTION
**Pull request by alexarchambault**
Fixes https://github.com/coursier/sbt-launcher/issues/27.

### CODE REVIEW

File: sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala [hunk 1]
```diff
@@ -1,363 +1,22 @@
...
+
+import caseapp.core.RemainingArgs
+
+object MainApp {
+  def main(args: Array[String]): Unit = {
+    val fromEnv = sys.env.get("COURSIER_SBT_LAUNCHER_PARSE_ARGS")
+      .map(s => s == "1" || s.toLowerCase(Locale.ROOT) == "true")
+    def fromProps = sys.props.get("coursier.sbt-launcher.parse-args")
+      .map(s => s == "1" || s.toLowerCase(Locale.ROOT) == "true")
+    val parseArgs = fromEnv.orElse(fromProps).getOrElse(false)
```
Comment (dwijnand) [inline sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala]: (minor) shouldn't sys props trump env vars?

Comment (alexarchambault) [reply sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala]: I've been doing the opposite in coursier. That allows users to use the environment to override whatever a script has set itself via a Java property.

Comment (dwijnand) [reply sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala]: Why can't a user override with a system property or with a command line option?

Comment (alexarchambault) [reply sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala]: That would work with sbt-extras, yes. But not from more convoluted scripts, that call coursier, or sbt-extras, on their own. (Although I guess it's even for that particular setting…)

Comment (dwijnand) [reply sbt-launcher/src/main/scala/coursier/sbtlauncher/MainApp.scala]: Yeah, I see the use case.  But I think it's backwards (and a common pitfall - sbt has a bunch of hidden sys prop usages).

It's probably best to go with this for now, and maybe later consider changing it holistically.

**Pull request merged**